microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -64,14 +65,21 @@ With this release, updates for both minor releases and patch releases are suppor
 //[id="microshift-4-14-support"]
 //=== Support
 
-//[id="microshift-4-14-post-installation"]
-//=== Post-installation configuration
+[id="microshift-4-14-post-installation"]
+=== Post-installation configuration
+
+[id="microshift-4-14-changes-config-yaml"]
+==== Configuration options changes
+* File placement for a custom YAML is now only `/etc/microshift/config.yaml`. `$HOME/.microshift/config.yaml` is no longer used.
```

Comment:
```suggestion
* File placement for a custom YAML is now only in `/etc/microshift/config.yaml`. `$HOME/.microshift/config.yaml` is no longer used.
```
Not sure if this is correct off the top of my head, but the sentence structure seemed strange.

Reply:
Yeah, it should say directory and not include the file itself. Fixing.